frontend/src/models/api/Document.ts

```diff
@@ -15,6 +17,7 @@ export interface Api_Document extends Api_ConcurrentVersion, Api_AuditFields {
 export interface Api_DocumentType extends Api_ConcurrentVersion, Api_AuditFields {
   id?: number;
   documentType?: string;
+  mayanId?: number;
```

Comment:
nit: would prefer to see this named "storageId"

Reply:
it could. The mayan* could be abstracted to be document storage